test/e2e/navigator/scenarios.js

```diff
@@ -0,0 +1,53 @@
+(function() {
+  'use strict';
+
+  describe('navigator', function() {
+    var path = '/test/e2e/navigator/index.html';
+    var EC = protractor.ExpectedConditions;
+
+    it('should exist', function() {
+      browser.get(path);
+      browser.waitForAngular();
+
+      var navi = element(by.css('ons-navigator'));
+
+      //Waits for the ons-navigator loading
+      browser.wait(EC.presenceOf(navi));
+
+      
+      expect(navi.isDisplayed()).toBeTruthy();
+    });
+
+    it('should have "var" attribute', function() {
+      browser.get(path);
+
+      var navi = element(by.css('ons-navigator'));
+
+      expect((navi.getAttribute('var')).isPresent()).toBeTruthy();
```

Comment:
I don't think this is working. Can you verify?

Also, there's a similar test for button.